app/views/docs/database.phtml

```diff
@@ -0,0 +1,216 @@
+</p>
+
+<h4>Collection Level Permissions</h4>
+<p>
+        With collection-level permissions, you assign permissions once for every document in the collection's settings - users with "read" access to the collection can see all documents in that collection. This permission model requires you to set collection permissions on the collection settings page. Document permissions are optional and will not be evaluated even when set.
+</p>
+
+<div class="notice margin-top-large margin-bottom-large">
+    <p>Note: For security purposes, an active Account session is required to create resources. So, even if wildcard write permissions are set, only logged-in users can create documents in a collection. If you require this behavior for your app, create an anonymous session first.</p>
+</div>
+
+
+<h2><a href="/docs/advanced-database#attributes" id="attributes">Create Attributes</a></h2>
+<p>
+    Once you choose your permission model, navigate to your collection's <b>Attributes</b> tab. Attributes are used to define the structure of your documents and help the Appwrite API validate your users' input. Add your first attribute by clicking the <b>Add Attribute</b> button. You can choose between the following types:
+</p>
+<table cellspacing="0" cellpadding="0" border="0" class="full margin-bottom-large">
+    <thead>
+        <tr>
+            <th style="width: 250px">Attribute</th>
+            <th>Description</th>
+        </tr>
+    </thead>
+    <tbody>
+        <tr>
+            <td>string</td>
+            <td>String attribute.</td>
+        </tr>
+        <tr>
+            <td>integer</td>
+            <td>Integer attribute.</td>
+        </tr>
+        <tr>
+            <td>float</td>
+            <td>Float attribute.</td>
+        </tr>
+        <tr>
+            <td>boolean</td>
+            <td>Boolean attribute.</td>
+        </tr>
+        <tr>
+            <td>enum</td>
+            <td>Enum attribute.</td>
+        </tr>
+        <tr>
+            <td>ip</td>
+            <td>IP address attribute for IPv4 and IPv6.</td>
+        </tr>
+        <tr>
+            <td>email</td>
+            <td>Email address attribute.</td>
+        </tr>
+        <tr>
+            <td>url</td>
+            <td>URL attribute.</td>
+        </tr>
+    </tbody>
+</table>
+
+<p>
+    If an attribute must be present in all documents, set it as <span class="tag">required</span>. If not, a <span class="tag">default</span> value might be handy. Additionally, decide if the attribute should be a primitive or array of values.
+</p>
+<p>
+    When adding or removing attributes, your requests are processed in the background, indicated by the attribute's <span class="tag">status</span>.
+    Depending on your collection's size and other factors, this could take anywhere from a few seconds to a few minutes to complete. Once all of your attributes are <span class="tag">available</span>, then you can create a document.
```

Comment:
```suggestion
    Depending on your collection's size and other factors, this could take anywhere from a few seconds to a few minutes to complete. You are able to create a document while your attributes are still being processed, however you are not able to use the attributes on the documents until they are <span class="tag">available</span>
```